lib/index.js

```diff
@@ -133,6 +133,10 @@ var Horseman = function Horseman(options) {
 		}
 	}
 
+    Object.keys(this.options.phantomOptions || {}).forEach(function (key) {
+		phantomOptions[key] = this.options.phantomOptions[key];
```

Comment:
I would like it to `debug` a warning here is one of horseman's options gets overridden (i.e. if `phantomOptions[key]` is already defined).